phabricator-user.rst

```diff
@@ -358,6 +358,31 @@ We will, however, display some revision metadata in associated
 bugs; see `bug 1489706
 <https://bugzilla.mozilla.org/show_bug.cgi?id=1489706>`_.
 
+Mozilla Phabricator Emails
+==========================
+
+.. image:: images/phab-email-new.png
+   :align: center
+   :alt: Screenshot of an example new Phabricator email
+
+There's a new Phabricator email system current in development
+with the following improvements over the existing system:
+
+* There's less noise per email.
```

Comment:
I'd probably expand on point 1 slightly and make point 3 slightly more formal. Suggestion:

* There's less noise per email, and fewer emails sent per event.
* Relevant contextual information is now more clearly available.
* They're faster to read and have less visual clutter than stock Phabricator emails.

Reply:
> They're faster to read and have less visual clutter than stock Phabricator emails.

IMHO, this feels like a duplicate of the first point: "there's less noise".
How about something like: "They're visually structured to be helpful and fast to read"?

Reply:
> How about something like: "They're visually structured to be helpful and fast to read"?

I like it :+1: